assets/css/phoenix.css

```diff
@@ -6,78 +6,113 @@
 
-/* General style */
-h1{font-size: 3.6rem; line-height: 1.25}
-h2{font-size: 2.8rem; line-height: 1.3}
-h3{font-size: 2.2rem; letter-spacing: -.08rem; line-height: 1.35}
-h4{font-size: 1.8rem; letter-spacing: -.05rem; line-height: 1.5}
-h5{font-size: 1.6rem; letter-spacing: 0; line-height: 1.4}
-h6{font-size: 1.4rem; letter-spacing: 0; line-height: 1.2}
-pre{padding: 1em;}
+ /* General style */
+ h1{font-size: 3.6rem; line-height: 1.25}
+ h2{font-size: 2.8rem; line-height: 1.3}
+ h3{font-size: 2.2rem; letter-spacing: -.08rem; line-height: 1.35}
+ h4{font-size: 1.8rem; letter-spacing: -.05rem; line-height: 1.5}
+ h5{font-size: 1.6rem; letter-spacing: 0; line-height: 1.4}
+ h6{font-size: 1.4rem; letter-spacing: 0; line-height: 1.2}
+ pre{padding: 1em;}
+ 
+ body{
```

Comment:
Can you lint your CSS? We typically put in a space between `body` and `{` for example.

Reply:
I'm not sure if this is a elixir/phoenix paradigm, but it looks as if the CSS is almost minified. It makes it a little difficult to read. I think you would typically see it written in this type of format:
```
h6 {
  font-size: 1.4rem;
  letter-spacing: 0;
  etc...
}
```

If this is how CSS is formatted when dealing with Elixir/Phoenix then please disregard.

Reply:
A follow-on to this would be it looks minified because this is the boilerplate that Phoenix provides? Would you typically change the CSS in here, or should there be a separate file or a separate section in this file so that you can keep track of what is stock and what you have edited? For organizational purposes?  I'm not telling you to do this, I'm just asking questions at this point :-D.

Reply:
It is formatted this way automatically by Phoenix, it's the boilerplate. I've been making changes in the un-minified code below, including cutting out some of their irrelevant css(or I'm planning to). Should  I have a separate file for my code?

Reply:
Yes, I would create a separate `main.css` rather than adding to the autogenerated CSS.